torch/_subclasses/fake_tensor.py

```diff
@@ -1233,14 +1233,20 @@ def dispatch(self, func, types, args=(), kwargs=None):
                 if op_impl_out != NotImplemented:
                     return op_impl_out
 
+        def can_fallback(func: OpOverload):
+            if not self.allow_fallback_kernels:
+                return False
+            # should't fallback on custom ops
+            return func.namespace in ["prim", "aten"]
```

Comment:
you mean prims, not prim. Also, we shouldn't allow fallback on prims anyway, they should ALWAYS have meta by construction